src/main/java/appeng/client/render/RenderBlocksWorkaround.java

```diff
@@ -626,6 +626,66 @@ public ISimplifiedBundle getLightingCache()
 		return new LightingCache( this.lightState );
 	}
 
+	public EnumSet<ForgeDirection> getFaces()
+	{
+		return this.faces;
+	}
+
+	public void setFaces( final EnumSet<ForgeDirection> faces )
+	{
+		this.faces = faces;
+	}
+
+	public boolean isCalculations()
+	{
+		return this.calculations;
+	}
+
+	public void setCalculations( final boolean calculations )
+	{
+		this.calculations = calculations;
+	}
+
+	public boolean isUseTextures()
+	{
+		return this.useTextures;
+	}
+
+	public void setUseTextures( final boolean useTextures )
+	{
+		this.useTextures = useTextures;
+	}
+
+	public boolean isFacade()
+	{
+		return this.isFacade;
+	}
+
+	public void setFacade( final boolean isFacade )
+	{
+		this.isFacade = isFacade;
+	}
+
+	public float getOpacity()
+	{
+		return this.opacity;
+	}
+
+	public void setOpacity( final float opacity )
+	{
+		this.opacity = opacity;
+	}
+
+	public EnumSet<ForgeDirection> getRenderFaces()
+	{
+		return this.renderFaces;
+	}
+
+	public void setRenderFaces( final EnumSet<ForgeDirection> renderFaces )
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The type of the "renderFaces" object should be an interface such as "Set" rather than the implementation "EnumSet". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1319)